src/bound/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod bound;
+
+pub use self::bound::*;
```

Comment:
usually you can move `<name>.rs` to `<name>/mod.rs` and it will feel a little more idiomatic.

For small stuff like bound, crc16, ops, radix, stack, I don't think there's much use splitting them into individual modules.